services/notifyUser.js

```diff
@@ -2,19 +2,19 @@ const { sendSms } = require('./sms/index')
 const { sendMail } = require('./email/index')
 
 function notifyUser(
-  isSmsAllowed,
-  userEmail,
+  user,
   subjectEmail,
   viewEmail,
-  userName,
   hotelName,
   from,
-  number,
   textSms
 ) {
-  sendMail(userEmail, subjectEmail, viewEmail, userName, hotelName)
+  console.log(user)
+  const { isSmsAllowed, email, firstName, lastName, phoneNumber } = user
+  const userFullName = `${firstName} ${lastName}`
```

Comment:
user has it already - fullName

Reply:
i couldnt see it in console

Reply:
this is virtual property, so probably you have to call it user.fullName